watchman/views.py

```diff
@@ -110,7 +114,10 @@ def dashboard(request):
 
     overall_status = all([type_status['ok'] for type_status in check_types])
 
-    return render(request, 'watchman/dashboard.html', {
+    response = render(request, 'watchman/dashboard.html', {
         'checks': check_types,
         'overall_status': overall_status
     })
+
+    response[WATCHMAN_VERSION_HEADER] = __version__
```

Comment:
Yeah, that's how they say to do it in the docs: https://docs.djangoproject.com/en/1.8/ref/request-response/#setting-header-fields